client/components/site-selector/index.jsx

```diff
@@ -104,19 +104,17 @@ class SiteSelector extends Component {
 			return;
 		}
 
-		if ( ! this.highlightedSiteRef ) {
-			selectorElement.scrollTop = 0;
+		const highlightedSiteElem = document.body.querySelector(
+			'.site-selector .site.is-highlighted, .site-selector .all-sites.is-highlighted'
```

Comment:
Shall we leave a note in render to update this if class names change?

Reply:
Great point. Done in cf65b5c.